application/jaxrs-cassandra-quarkus/src/test/java/webapp/tier/service/CassandraServiceTest.java

```diff
@@ -0,0 +1,117 @@
+package webapp.tier.service;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import com.datastax.oss.quarkus.test.CassandraTestResource;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import webapp.tier.bean.MsgBean;
+
+@QuarkusTest
+@QuarkusTestResource(CassandraTestResource.class)
+class CassandraServiceTest {
+
+	@Inject
+	CassandraService svc;
+
+    @AfterEach
+    void afterEach() {
+    	svc.deleteMsg();
+    }
+
+	String respbody = "message: Hello k8s-3tier-webapp with quarkus";
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/yurake/k8s-3tier-webapp/pullRequest?prid=6394994)